deployment_test.go

```diff
@@ -37,6 +37,23 @@ func TestDeployments(t *testing.T) {
 	assert.Equal(t, len(deployment.Steps), 1)
 }
 
+func TestDeploymentsV1(t *testing.T) {
+	cfg := NewDefaultConfig()
+	cfg.Version = "v1.1.1"
```

Comment:
nit-pick: I'd remove the `v` prefix. It should be clear that field contains numeric version specifiers only.